src/DataStructures/DataBox/DataBox.hpp

```diff
@@ -1792,6 +1839,40 @@ inline constexpr auto mutate_apply(
                                       ArgumentTags{},
                                       std::forward<Args>(args)...);
 }
+// @}
+
+/*!
+ * \ingroup DataBoxGroup
+ * \brief Apply the stateless function `F` mutating its `return_tags` and taking
```

Comment:
`F` is not a function, it's a class or type that is used to generate a closure which then has its `apply` function invoked.